test/cmactest.c

```diff
@@ -77,7 +77,7 @@ static struct test_st {
             0x0b, 0x0c, 0x0d, 0x0e, 0x0f
         },
         16,
-        /* fullfill data to 3072 bytes with below string */
+        /* fulfill data to 3072 bytes with below string */
```

Comment:
`fulfill` isn't the right word, so if you're going to change it you might as well use a better word

Reply:
I was thinking of something a bit more descriptive, maybe

```
/* String below is repeated to give the number of bytes of data (3072) */
```